index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css?family=Poppins:100,100i,200,200i,300,300i,400,400i,500,500i,600,600i,700,700i,800,800i,900,900i&amp;display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+
+    <title>PortFolio</title>
+</head>
+<body>
+   <header>
+        <div class="container">
+            <p id="logo">WELCOM</p>
+            <div class="menu"><i class="fa fa-bars"></i></div>
+        </div>
+
+   </header>
+   <main>
+            <h2>I’m Maaloum <br>
+                Glad to see you!</h2>
+            <p class ="description">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="connect">LET'S CONNECT</a>
+            <div class="contacts">
+                <a href="#" class ="link"><i class="fa fa-twitter"></i></a>
+                <a href="#" class ="link"><i class="fa fa-linkedin-square" ></i></a>
+                <a href="#" class ="link"><i class="fa fa-github"></i></a>
+                <a href="#" class ="link"><i class= 'fa fa-hand-peace-o'></i></a>
+                <a href="" class="link"><i class="material-icons">feedback</i></a>
+            </div>
```

Comment:
- [ ] Since these are lists of icons, it will be clearer if you could please wrap them in a `ul` and a `li` for each icon. 

See example:
```html
<ul>
   <li><a href="..."> // your icon </a></li>
   <li><a href="..."> // your icon </a></li>
</ul>
```